frontend/src/views/Layout/DashboardLayout_gdd.vue

```diff
@@ -23,7 +23,6 @@
       </template>
     </side-bar>
     <div class="main-content">
-      <dashboard-navbar :type="$route.meta.navbarType"></dashboard-navbar>
```

Comment:
This is broken!

![image](https://user-images.githubusercontent.com/15882241/121815148-38232780-cc75-11eb-8586-1dbf42831d09.png)

In desktop view we have no top bar anymore. The sidebar looks bad